bvm/ballerina-runtime/src/main/java/org/ballerinalang/jvm/util/RuntimeUtils.java

```diff
@@ -137,6 +137,14 @@ public static void handleRuntimeErrors(Throwable throwable) {
         Runtime.getRuntime().exit(1);
     }
 
+    public static void handleRuntimeErrorWithoutShutDown(Throwable throwable) {
```

Comment:
IMO better way is to use handleRuntimeErrors without Runtime.getRuntime().exit(1) and add a new method "handleRuntimeErrorsAndExit" which calls handleRuntimeErrors +  Runtime.getRuntime().exit(1). In  that case you need to change the current usages in handleRuntimeErrors and   HANDLE_THROWABLE_METHOD constant in jvmconstants